WebApp/autoreduce_webapp/templates/instrument_variables.html

```diff
@@ -19,109 +18,135 @@ <h2>{{ instrument.name }} - Configure New Jobs</h2>
+                      class="form-horizontal">
             {% else %}
-                <form id="instrument:variables" method="POST" action="{% url 'instrument:variables' instrument=instrument.name start=run_start end=run_end %}" class="form-horizontal">
+                <form id="instrument_variables" method="POST"
+                      action="{% url 'instrument:variables' instrument=instrument.name start=run_start end=run_end %}"
+                      class="form-horizontal">
             {% endif %}
-                {% csrf_token %}
-                <input type="hidden" id="upcoming_runs" name="upcoming_runs" value="{{upcoming_run_variables}}" />
-                <input type="hidden" id="is_editing" name="is_editing" value="{{editing}}" />
-                <div class="row">
-                    <div class="col-md-9">
-                        {% include "snippets/form_warnings.html" %}
-                        <div class="text-center">
-                            <span class="js-experiment-label">By Experiment Reference Number</span>
-                            <input type="checkbox" name="variable-range-toggle" id="variable-range-toggle" data-on-text="&nbsp;" data-off-text="&nbsp;" data-off-color="default"  data-on-color="default" data-animate="false" data-invert="true" />
-                            <input type="hidden" name="variable-range-toggle-value" id="variable-range-toggle-value" value="{% if experiment_reference > 0 %}False{% else %}True{% endif %}" />
-                            <span class="js-run-label">By Run Number Range</span>
-                        </div>
-                        <br/>
-                        <div class="js-variable-by-experiment">
-                            <div class="row">
-                                <div class="col-md-12">
-                                    <div class="form-group">
-                                        <label for="run_start" class="control-label col-md-3">Experiment Reference Number</label>
-                                        <div class="col-md-9">
-                                            <input type="number" id="experiment_reference_number" name="experiment_reference_number" value="{{experiment_reference}}" class="form-control"/>
-                                        </div>
-                                    </div>
+        {% csrf_token %}
+        <input type="hidden" id="upcoming_runs" name="upcoming_runs" value="{{ upcoming_run_variables }}"/>
+        <input type="hidden" id="is_editing" name="is_editing" value="{{ editing }}"/>
+        <div class="row">
+            <div class="col-md-9">
+                {% include "snippets/form_warnings.html" %}
+                <div class="text-center">
+                    <span class="js-experiment-label">By Experiment Reference Number</span>
+                    <input type="checkbox" name="variable-range-toggle" id="variable-range-toggle" data-on-text="&nbsp;"
```

Comment:
Are inputs without associated labels an accessibility issue?
And in a few places below on this template

Reply:
I decided not to work on anything involving instrument variables because of this concurrent PR https://github.com/ISISScientificComputing/autoreduce/pull/1094 but a agree that's an accessibility issue.